docs/api/app.md

```diff
@@ -1302,23 +1302,24 @@ Returns `Object`:
 Set the app's login item settings.
 
 To work with Electron's `autoUpdater` on Windows, which uses [Squirrel][Squirrel-Windows],
-you'll want to set the launch path to Update.exe, and pass arguments that specify your
-application name. For example:
+you'll want to set the launch path to your executable's name but a directory up, which is
+a stub application automatically generated by Squirrel which will automatically launch the
+latest version.
 
 ``` js
 const { app } = require('electron')
 const path = require('node:path')
 
 const appFolder = path.dirname(process.execPath)
-const updateExe = path.resolve(appFolder, '..', 'Update.exe')
-const exeName = path.basename(process.execPath)
+const ourExeName = path.basename(process.execPath)
+const stubLauncher = path.resolve(appFolder, '..', ourExeName)
 
 app.setLoginItemSettings({
   openAtLogin: true,
-  path: updateExe,
+  path: stubLauncher,
```

Comment:
I think quotes might be needed in case the installation path contains spaces.
```suggestion
  path: `"${stubLauncher}"`,
```

Looking at my registry's entries for `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run`, most applications wrap their path in quotes, but not all.

Reply:
I feel like if Electron requires this then this is a bug in Electron not what we should be encouraging in the docs

Reply:
Given we previously did not wrap the value for `path`, that's fair. Agreed that this should be fixed within Electron.